### PR TITLE
Create Gas Timing component for EIP-1559

### PR DESCRIPTION
**Pull request by darkwing**
GasTimings

We're going to be using this block in multiple places, so I created a separate component to ensure consistency.

### CODE REVIEW

File: ui/components/app/gas-timing/gas-timing.component.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      })}
+    >
+      {text}
+      {tooltipText && <InfoTooltip position="top" contentText={tooltipText} />}
+    </Typography>
+  );
+}
+
+GasTiming.propTypes = {
+  text: PropTypes.string,
```
Comment (dan437) [inline ui/components/app/gas-timing/gas-timing.component.js:24]: For props that are required, can we use the `.isRequired` prop check? https://reactjs.org/docs/typechecking-with-proptypes.html

In this case should be at least this one:
```
text: PropTypes.string.isRequired,
```

File: ui/components/app/gas-timing/gas-timing.stories.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export const simple = () => {
+  return (
+    <div style={{ width: '600px' }}>
+      <GasTiming text="Likely within 30 seconds" />
+    </div>
+  );
+};
+
+export const withTooltip = () => {
```
Comment (dan437) [inline ui/components/app/gas-timing/gas-timing.stories.js:17]: I really like to see Storybook files for reusable components!

**Review by dan437 — DISMISSED** [review]
Looking forward to using this component!

**Review by danjm — DISMISSED** [review]
Looks good. Just need to address the request to add `.isRequired` to appropriate component properties.

**Review by brad-decker — DISMISSED** [review]

**Review by dan437 — DISMISSED** [review]

**Review by dan437 — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**